packages/ui-popover/src/Popover/index.tsx

```diff
@@ -565,62 +512,56 @@ class Popover extends Component<PopoverProps> {
     }
 
     if (this.shown || this.props.shouldRenderOffscreen) {
-      let ViewElement
       const color = this.props.color
 
-      let viewProps = {
-        // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'c' implicitly has an 'any' type.
-        ref: (c) => (this._view = c),
-        // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'el' implicitly has an 'any' type.
-        elementRef: (el) => {
-          // @ts-expect-error ts-migrate(2339) FIXME: Property '_contentElement' does not exist on type ... Remove this comment to see the full error message
+      let viewProps: (Partial<ViewProps> | Partial<ContextViewProps>) & {
```

Comment:
I made a little refactor here too. It isn't easy to type, because this can be either the props of View of ContextView, and can return View os ContextView.